keyboards/orthodox/keymaps/xyverz/config.h

```diff
@@ -22,7 +22,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #ifndef CONFIG_USER_H
```

Comment:
We've begun recommending the use of `#pragma once` include guards instead of the old style `#ifndef`-`#define`-`#endif` method you're using here.

Replace this line, and then delete Lines 23 and 38.

```suggestion
#pragma once
```

Reply:
Thanks!